com.github.giacomogroppi.writernote-qt.json

```
@@ -0,0 +1,88 @@
+{
+  "app-id": "com.github.giacomogroppi.writernote-qt",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "6.2",
+  "sdk": "org.kde.Sdk",
+  "command": "writernote",
+  "finish-args": [
+    "--socket=wayland",
+    "--socket=x11",
```

```suggestion
    "--socket=wayland",
    "--socket=fallback-x11",
```

assuming it supports wayland properly.